src/coreclr/vm/peimage.inl

```diff
@@ -36,43 +36,67 @@ inline const SString& PEImage::GetPathToLoad()
 {
     LIMITED_METHOD_DAC_CONTRACT;
 
-    return IsInBundle() ? m_bundleFileLocation.Path() : m_path;
+    return IsInBundle() ? m_probeExtensionResult.BundleLocation.Path() : m_path;
 }
 
 inline void* PEImage::GetExternalData(INT64* size)
 {
     LIMITED_METHOD_CONTRACT;
 
     _ASSERTE(size != nullptr);
+    if (!IsExternalData())
+    {
+        *size = 0;
+        return nullptr;
+    }
 
-    *size = m_bundleFileLocation.Size;
-    return m_bundleFileLocation.DataStart;
+    *size = m_probeExtensionResult.ExternalData.Size;
+    return m_probeExtensionResult.ExternalData.Data;
 }
 
 inline INT64 PEImage::GetOffset() const
 {
     LIMITED_METHOD_CONTRACT;
 
-    return m_bundleFileLocation.Offset;
+    return IsInBundle() ? m_probeExtensionResult.BundleLocation.Offset : 0;
 }
 
 inline BOOL PEImage::IsInBundle() const
 {
     LIMITED_METHOD_CONTRACT;
 
-    return m_bundleFileLocation.IsValid();
+    return m_probeExtensionResult.Type == ProbeExtensionResult::Type::Bundle;
 }
 
-inline INT64 PEImage::GetSize() const
+inline BOOL PEImage::IsExternalData() const
 {
     LIMITED_METHOD_CONTRACT;
-    return m_bundleFileLocation.Size;
+
+    return m_probeExtensionResult.Type == ProbeExtensionResult::Type::External;
 }
 
-inline INT64 PEImage::GetUncompressedSize() const
+inline INT64 PEImage::GetSize() const
+{
+    if (!m_probeExtensionResult.IsValid())
```

Comment:
This is definitely not what you want.

Reply:
Oh, I had fixed that. Must not have pushed.